packages/markdown/src/lib/MarkdownPlugin.ts

```diff
@@ -43,6 +54,7 @@ export const MarkdownPlugin = createTSlatePlugin<MarkdownConfig>({
   options: {
     elementRules: remarkDefaultElementRules,
     indentList: false,
+    keepLineBreak: false,
```

Comment:
I think the "keep" in the option name is misleading. "Keep" would imply that setting the option to true preserves line breaks in their natural form. Maybe we could call the option `splitAtLineBreaks`?